librz/arch/analysis.c

```diff
@@ -67,8 +67,25 @@ static void meta_item_free(void *item) {
 	free(it);
 }
 
-RZ_API RzAnalysis *rz_analysis_new(void) {
+RZ_API RzAnalysis *rz_analysis_new(RZ_BORROW RZ_NULLABLE const char *sys_path_prefix) {
```

Comment:
```suggestion
RZ_API RzAnalysis *rz_analysis_new(RZ_NULLABLE const char *sdb_types_path) {
```

Reply:
Borrow is not needed if `const char*` is passed. just make sure to copy the string via `rz_str_dup(sdb_types_path)`